### ISSUE TITLE
XMPP matrix.org bridge: events only passing from XMPP to Matrix, not vice versa

### ISSUE THREAD

**Issue by alphapapa** [matrix-org/matrix-bifrost#338]
Forgive me if this isn't the proper place to report this problem.  Please direct me to a more appropriate place if there is one.  I've already asked in `#bifrost:half-shot.uk` but it looks like the room isn't monitored.

I've been in some XMPP rooms bridged through the matrix.org bridge, both direct rooms and public ones, for several months, and they've been working flawlessly.  But since a couple of days ago, it seems that messages are only passing in one direction: I receive messages from XMPP users, but the XMPP users aren't seeing my messages anymore.  https://status.matrix.org shows no downtime on the XMPP bridge.  No errors are encountered; from Matrix's perspective, my messages are in the rooms, it's just that the XMPP users aren't receiving them.

cc: @josephmturner

**Comment by alphapapa** [comment]
Since I've received no answer in the room or here, I just emailed support@matrix.org.  I don't know what else to do or who to try to contact to report the problem.

**Comment by Neustradamus** [comment]
To follow this important ticket, thanks @alphapapa.

**Comment by Neustradamus** [comment]
@alphapapa, @hanthor, @mirabilos: ProcessOne has announced Matrix support in ejabberd: https://www.process-one.net/blog/matrix-protocol-added-to-ejabberd/

cc: @mremond.

**Comment by mirabilos** [comment]
Neustradamus dixit:

>@alphapapa, @hanthor, @mirabilos: ProcessOne has announced Matrix
>@support in ejabberd:

Won’t help m̲e̲… :/ I need this bridge fixed.

bye,
//mirabilos
-- 
Support mksh as /bin/sh and RoQA dash NOW!
‣ src:bash (429 (458) bugs: 0 RC, 295 (315) I&N, 134 (143) M&W, 0 F&P) + 208
‣ src:dash (82 (96) bugs: 0 RC, 44 (47) I&N, 38 (49) M&W, 0 F&P) + 62 ubu
‣ src:mksh (1 bug: 0 RC, 0 I&N, 1 M&W, 0 F&P)
dash has two RC bugs they just closed because they don’t care about quality…

**Comment by Neustradamus** [comment]
@mirabilos: Yes a lot people would like a nice bridge since several years...

**Comment by mirabilos** [comment]
Neustradamus dixit:

>@mirabilos: Yes a lot people would like a nice bridge since several years...

What I meant is, making that just for ejabberd users won’t help.

bye,
//mirabilos
-- 
  "Using Lynx is like wearing a really good pair of shades: cuts out
   the glare and harmful UV (ultra-vanity), and you feel so-o-o COOL."
                                         -- Henry Nelson, March 1999

**Comment by alphapapa** [comment]
> @alphapapa, @hanthor, @mirabilos: ProcessOne has announced Matrix support in ejabberd: https://www.process-one.net/blog/matrix-protocol-added-to-ejabberd/
> 
> cc: @mremond.

Good news, indeed, although that announcement was made almost a year ago.  :)  As well, it says that Matrix support would initially only be available in their commercial products before later reaching the community edition.  I wonder what the current status is.

**Comment by Neustradamus** [comment]
@alphapapa: I do not know, I would like to know too ^^

**Comment by alphapapa** [comment]
FWIW, I've found the aria-net.org Bifrost bridge to be working well with XMPP: https://aria-net.org/SitePages/Portal/Bridges.aspx

**Comment by Neustradamus** [comment]
@alphapapa: Yes it is a fork by the author of Metronome IM (which is based on Prosody IM), @maranda, a good developer:
- https://github.com/matrix-org/matrix-bifrost/issues?q=is%3Aissue+author%3Amaranda+
- https://github.com/matrix-org/matrix-bifrost/pulls?q=is%3Apr+author%3Amaranda

**Comment by Samgarr** [comment]
It looks like I have the same problem, how can I help with debugging?
